hi-ml-histopathology/src/histopathology/configs/classification/BaseMIL.py

```diff
@@ -92,6 +96,15 @@ def setup(self) -> None:
         self.encoder = self.get_encoder()
         if not self.is_finetune:
             self.encoder.eval()
+        # Fine-tuning requires tiles to be loaded on-the-fly, hence, caching is disabled by default.
```

Comment:
[optional] Consider moving the configuration below into `__init__()` to ensure that these parameters are correctly set as soon as the container is created.
```python
class BaseMIL:
  def __init__(self, **kwargs):
    super().__init__(**kwargs)
    if self.is_finetune:
      ...
```

Reply:
I've reverted this as per our discussion.